### PR TITLE
Display tokens and addresses in Overview page

### PR DESCRIPTION
**Pull request by nop33**
Some basic stuff

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
   },
   "devDependencies": {
     "@alephium/sdk": "0.5.0",
+    "@alephium/tokens-list": "file:../tokens-meta/alephium-tokens-list-0.0.0.tgz",
```
Comment (nop33) [inline package.json:37]: The package is not yet published and is in a private GH repo

Comment (mvaivre) [reply package.json:37]: Where can I get this tarball?

Comment (nop33) [reply package.json:37]: you can build it by checking out the master branch of the project https://github.com/alephium/tokens-meta and running

```
nvm use
npm i
npm run build
npm pack
```

Comment (nop33) [reply package.json]: Actually, you don't have to do that anymore! I published the package and updated this PR.

**Comment by nop33** [comment]
@mvaivre, just a kind ping that the review is pending :)

**Comment by nop33** [comment]
Superseded by #529. Closing PR.

**Pull request closed without merging**